scripts/bountiful.ash

```diff
@@ -335,6 +335,12 @@ string addBountyToQueue(monster opp, boolean speculate) {
     return "skill Motif";
   }
 
+  //Why not Red White and Blue?
+  if(!have_effect($effect[Everything Looks Red, White and Blue]) && have_skill($skill[%fn, fire a Red White and Blue Blast]))
```

Comment:
`!have_effect($effect[Everything Looks Red, White and Blue])`
should be
`have_effect($effect[Everything Looks Red, White and Blue]) == 0`

not intuitive, but `have_effect` returns an int, not a boolean

Reply:
did this in the cli
`ash $skill[%fn, fire a Red White and Blue Blast]`

It suggested:
`%fn\, fire a Red\, White and Blue Blast`

Updating to that would remove some nuance messages from mafia

Reply:
Should I change the 'return' as well?

Reply:
No that one should be fine

Reply:
Excellent. Updated.